### ISSUE TITLE
[Fix in #5264] Missing visual focus on Search box

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4293]
![Screen Shot 2019-08-28 at 9 59 34 PM](https://user-images.githubusercontent.com/15113219/63904589-90753080-c9df-11e9-9b70-3fb19ed455fb.png)

Should there be focus on the magnifying glass as well?

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
